tests/core/src/main/java/bin/sample-runbt.sh

```diff
@@ -78,11 +78,20 @@ list=`ls $GEMFIRE/lib/gemfirexd-client-*`
 filename=`tr "/" "\n" <<< $list | tail -1`
 
 export releaseVersion=`echo "${filename%*.*}"| cut -d'-' -f3-4`
-export CLASSPATH=$JTESTS:$EXTRA_JTESTS:$GEMFIRE/lib/gemfirexd-${releaseVersion}.jar:$GEMFIRE/lib/gemfirexd-client-${releaseVersion}.jar:$JTESTS/../../libs/gemfirexd-hydra-tests-${releaseVersion}-all.jar:$GEMFIRE/lib/gemfirexd-tools-${releaseVersion}.jar
+
+assemblyJarName=`ls $GEMFIRE/../snappy/lib/snappydata-assembly*`
+assemblyJarFilename=`tr "/" "\n" <<< $assemblyJarName | tail -1`
+export assemblyJarVersion=`echo "${assemblyJarFilename%*.*}"| cut -d'_' -f2`
+
+snappyTestsJarName=`ls $SNAPPYDATADIR/dtests/build-artifacts/scala-2.10/libs/gemfirexd-scala-tests*`
+snappyTestsJarFilename=`tr "/" "\n" <<< $snappyTestsJarName | tail -1`
+export snappyTestsJarVersion=`echo "${snappyTestsJarFilename%*.*}"| cut -d'-' -f4-6`
+
+export CLASSPATH=$JTESTS:$EXTRA_JTESTS:$GEMFIRE/lib/gemfirexd-${releaseVersion}.jar:$GEMFIRE/lib/gemfirexd-client-${releaseVersion}.jar:$JTESTS/../../libs/gemfirexd-hydra-tests-${releaseVersion}-all.jar:$GEMFIRE/lib/gemfirexd-tools-${releaseVersion}.jar:$SNAPPYDATADIR/dtests/build-artifacts/scala-2.10/libs/gemfirexd-scala-tests-${snappyTestsJarVersion}.jar:$GEMFIRE/../snappy/lib/snappydata-assembly_${assemblyJarVersion}.jar
```

Comment:
@swatisawant With snappydata-assembly jar in CLASSPATH you should not need gemfirexd/gemfirexd-client/gemfirexd-tools jars. I will recommend removing them since that is the way users will have our clusters running.